kafka/tools/assigner.py

```diff
@@ -570,217 +570,223 @@ def module_balance_even(cluster, args):
 
   return cluster
 
-########################################################################################################################
```

Comment:
Note that this change is indentation, the removal of the "main" comment, and the addition of a main function for use in setup.py.